lr_face/models.py

```diff
@@ -33,8 +33,11 @@ class Deepface_Lib_Model:
     deepface/Face model
     """
 
-    def __init__(self, model):
-        self.model = model
+    def __init__(self, module):
+        self.module = module
+        # lazy loading
+        self.model = None
```

Comment:
Makkelijker om van `self.model` dan een `@property` te maken ipv een attribuut denk ik? Stel dat je later nog een functie toevoegt waar je het model nodig hebt, dan moet je in al die functies `if not self.model: self.model = self.module.loadModel()` gaan typen.

```python
@property
def model(self):
    if not self._model:
        self._model = self.module.loadModel()
    return self._model
```

Moet je alleen even `self._model` op `None` zetten in de `__init__()`

Reply:
mooier, thanks!